packages/lesswrong/server/migrations/20221221T222213.auto.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Generated on 2022-12-21T22:22:13.062Z by `yarn makemigrations`
+ * The following schema changes were detected:
+ * -------------------------------------------
+ * ***Diff too large to display***
+ * -------------------------------------------
+ * (run `git diff --no-index schema/accepted_schema.sql schema/schema_to_accept.sql` to see this more clearly)
+ *
+ * - [ ] Write a migration to represent these changes
+ * - [ ] Rename this file to something more readable if you wish
+ * - [ ] Uncomment `acceptsSchemaHash` below
+ * - [ ] Run `yarn acceptmigrations` to update the accepted schema hash (running makemigrations again will also do this)
+ */
+// export const acceptsSchemaHash = "56ffdafb618c6041d2fc772be08e6e06";
+
+export const up = async ({db}: MigrationContext) => {
+  // TODO
```

Comment:
TODO?

Reply:
Autogenerated file that I don't want to deal with.